osf/management/commands/activate_sloan_flag.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from django.core.management.base import BaseCommand
+from waffle.models import Flag
+
+logger = logging.getLogger(__name__)
+
+from osf.features import (
+    SLOAN_DATA,
+    SLOAN_PREREG,
+    SLOAN_COI
+)
+
+SLOAN_FLAGS = [
+    SLOAN_DATA,
+    SLOAN_PREREG,
+    SLOAN_COI
+]
+
+
+def activate_single_flag(flag_name, value):
+    # Only on flag at a time.
+    Flag.objects.filter(name__in=SLOAN_FLAGS).update(everyone=False)
+
+    flag = Flag.objects.get(name=flag_name)
+    flag.everyone = value
+    flag.save()
+
+
+class Command(BaseCommand):
+    """Flips sloan flags on and off, ensures only one flag is active at any given time.
```

Comment:
Wouldn't they just set this in the admin app? Also, aren't we keeping active any completed test items when we move on to the next test?

Reply:
@brianjgeiger I vaguely remember you saying there should be a management command for this, potentially I was thinking of something else?

Reply:
Maybe for waffle flags in general? There's a [ticket to move to adding waffle flags by management command](https://openscience.atlassian.net/browse/ENG-1410) rather than migration, because unnecessary migrations are a pain to manage and also cause deployment troubles.